ethcore/src/miner/miner.rs

```diff
@@ -1695,6 +1695,30 @@ mod tests {
 		assert_eq!(miner.prepare_pending_block(&client), BlockPreparationStatus::NotPrepared);
 	}
 
+	#[test]
+	fn should_reject_local_transaction_with_invalid_chain_id() {
+		let spec = spec::new_test();
+		let miner = Miner::new_for_tests(&spec, None);
+		let client = TestBlockChainClient::default();
+		let chain_id = spec.chain_id();
+
+		// chain_id + 100500 is invalid
+		let import = miner.import_claimed_local_transaction(
+			&client,
+			PendingTransaction::new(transaction_with_chain_id(chain_id + 10500), None), 
```

Comment:
```suggestion
			PendingTransaction::new(transaction_with_chain_id(chain_id + 10500), None),
```

I hate to be that guy, but trailing whitespaces :P